ansible/configs/ansible-middleware-workshop/post_software.yml

```diff
@@ -0,0 +1,85 @@
+---
+
+
+- name: Step 00xxxxx post software
+  hosts: localhost
+  gather_facts: false
+  become: true
+  tasks:
+    - debug:
+        msg: "Post-Software tasks Started"
+
+- name: Deploy chrome on bastion
+  hosts: bastions
+  gather_facts: false
+  become: true
+  tasks:
+    - name: Install the chrome rpm from a remote repo
+      yum:
+        name: https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
+        disable_gpg_check: true
+        state: present
+
+- name: Deploy user setup
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  tags:
+    - step005
+  tasks:
+    - name: osp user.info
+      when: cloud_provider == 'osp'
+      block:
+        - name: print out user.info
+          agnosticd_user_info:
+            msg: "{{ item }}"
+          loop:
+            - "To access bastion via SSH:"
+            - "ssh {{ student_name }}@bastion.{{ guid }}.{{ osp_cluster_dns_zone }}"
```

Comment:
It would be good if we could start moving to using requester_username and requester_email in AgnosticD rather than student_name and email as the user is often not a student and the variable email isn't well qualified and is likely to cause surprises and pain for developers.
              guid: "{{ guid }}"
              owner_name: "{{ student_name }}"
              env_type: "{{ env_type | default('') }}"
              owner_email: "{{ email }}"
              note: "{{ sandbox_note | default('') }}"
              cloud_provider: "{{ cloud_provider }}"